src/test/java/test/tesng1046/TestCustomNamesForTests.java

```diff
@@ -0,0 +1,33 @@
+package test.tesng1046;
```

Comment:
@juherr - Thanks for pointing it out. Didn't realise it. I will take care of this as well.